roles/generate_manifests/tasks/main.yml

```diff
@@ -38,6 +38,16 @@
     - agent-config.yaml.j2
     - install-config.yaml.j2
 
+# The agent-config files are eventually deleted after its usage, just
+# copy them in a safe place in case of needing to check them
+- name: Save a backup of agent-config files
+  copy:
```

Comment:
This should probably be a fetch task and put it into the fetched dir. Like [this task](https://github.com/redhat-partner-solutions/crucible/blob/main/roles/generate_ssh_key_pair/tasks/main.yml#L28)

Reply:
updated